cms/static/karma_cms_squire.conf.js

```diff
@@ -0,0 +1,113 @@
+// Common JavaScript tests, using RequireJS.
+//
+// To run all the tests and print results to the console:
+//
+//   karma start cms/static/karma_cms_squire.conf.js
+//
+//
+// To run the tests for debugging: Debugging can be done in any browser
+// but Chrome's developer console debugging experience is best.
+//
+//   karma start cms/static/karma_cms_squire.conf.js --browsers=BROWSER --single-run=false
+//
+//
+// To run the tests with coverage and junit reports:
+//
+//   karma start cms/static/karma_cms_squire.conf.js --browsers=BROWSER --coverage
+// --junitreportpath=<xunit_report_path> --coveragereportpath=<report_path>
+//
+// where `BROWSER` could be Chrome or Firefox.
+//
+
+/* jshint node: true */
+/*jshint -W079 */
+'use strict';
+var path = require('path');
+var _ = require('underscore');
+var configModule = require(path.join(__dirname, '../../common/static/common/js/karma.common.conf.js'));
+
+var files = [
+    {pattern: 'xmodule_js/common_static/js/vendor/requirejs/require.js', included: false},
```

Comment:
It would be great if this config file could load the non-Squire version, so that we don't have to define all the dependencies twice.

We do this in the UI Toolkit like this:

https://github.com/edx/edx-ui-toolkit/blob/master/test/karma.ci.conf.js